src/classes/HealthCheckGroupController.cls

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
I am really curious. This class is in 230 already. Why this PR is trying to add it as a new class again?

Reply:
That's a good question. It looks like we had a hiccup with feature/230

Reply:
It looks like it's resolved now